### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by ascott1**
This makes several improvements as suggested by our UX team.

## Changes

- Make sure all text on a gray background is accessible
- Move effective date placement so that it is in line with the regulation text
- Opens all "what's this" links in a new tab
- Defaults the SxS analysis sidebar drawer to open when it contains content
- Adds bullet points to the reg landing page
- Makes the call to action on the landing page a button (instead of a standard text link)
- Moves the "new effective date" call out to the main content area

## Review

- @willbarton and/or @KimberlyMunoz 

## Screenshots

screen shot 2015-08-13 at 12 09 08 pm

screen shot 2015-08-13 at 12 09 46 pm

### CODE REVIEW

**Comment by ascott1** [comment]
All of the button code was part of an internal template. That commit has been reverted. This can be merged once it's reviewed @willbarton 

screen shot 2015-08-14 at 11 25 11 am

**Comment by willbarton** [comment]
So, the spacing to the left of "Subpart A" in the first screenshot not matching the space to the right of "Effective Date" bugs me, visually. Should it have the same right alignment over the body content?

**Comment by ascott1** [comment]
> So, the spacing to the left of "Subpart A" in the first screenshot not matching the space to the right of "Effective Date" bugs me, visually. Should it have the same right alignment over the body content?

I agree that it probably should. That's how it was mocked up, but let me follow up on it

**Comment by willbarton** [comment]
Another comment, the bullet points on landing pages line up to the left of the header. In an of itself, I don't have a problem with that... but it's visually jarring because the ordered lists we use for regulation text have their markers line up directly under their heading.

**Comment by ascott1** [comment]
Oh, hmm. Looking at the design manual, the bullets align with the text. Going to make that change. Good catch @willbarton!

**Comment by ascott1** [comment]
@willbarton these should be fixed. Thanks for being a developer/typographical-wizard now that "eagle eyes" @jehlers is no longer with us.

Effective date aligns with the interpreation box:

screen shot 2015-08-18 at 10 23 46 am 


Bullets are correctly indented:

screen shot 2015-08-18 at 10 24 21 am

**Comment by willbarton** [comment]
:+1: This looks good to me.

@KimberlyMunoz Do you want to take a pass at it before it's merged?

**Comment by KimberlyMunoz** [comment]
I can do that. I just got caught up with all my email and stuff. :)

File: regulations/static/regulations/css/less/layout.less [hunk 1]
```diff
@@ -187,24 +187,36 @@ TOC Header Navigation
...
+            margin-left: 40px;
+        }
+    }
 }
 
 .header-secondary {
-    float: right;
-    padding-right: 15px;
-    margin-right: -4px;
+    //float: right;
```
Comment (KimberlyMunoz) [inline regulations/static/regulations/css/less/layout.less]: should this be removed?

**Comment by KimberlyMunoz** [comment]
:+1:

File: regulations/static/regulations/css/less/layout.less (hunk 1 above)
Comment (ascott1) [inline regulations/static/regulations/css/less/layout.less]: Yes, it should!

**Pull request merged**